x-pack/test/api_integration/apis/monitoring/es_archives/_health/metricbeat_8/mappings.json

```diff
@@ -0,0 +1,374 @@
+{
+  "type": "index",
```

Comment:
I would expect `metricbeat-8.x.x` to be a datastream, did we generate this archive with the esArchiver ?

Reply:
Honestly, I just realized there was such tool after your comment. I just ran it and it did generate a mappings.json with "type": "data_stream". I'll update this.

Reply:
I think I have some documentation stashed about it, I'll try to revive it